src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs

```diff
@@ -573,45 +573,8 @@ public bool OnEditorAction(TextView v, [GeneratedEnum] ImeAction actionId, KeyEv
 		}
 
 		partial void OnTextCharacterCasingChangedPartial(DependencyPropertyChangedEventArgs e)
-		{
-			if (_textBoxView == null)
-			{
-				return;
-			}
-
-			var casing = (CharacterCasing)e.NewValue;
-
-			UpdateCasing(casing);
-		}
-
-		private void UpdateCasing(CharacterCasing characterCasing)
-		{
-			var currentFilters = _textBoxView.GetFilters()?.ToList() ?? new List<IInputFilter>();
-
-			//Remove any casing filters before applying new ones.
-			currentFilters.Remove(a => a is InputFilterAllLower || a is InputFilterAllCaps);
-
-			switch (characterCasing)
-			{
-				case CharacterCasing.Lower:
-					var lowerFilter = new List<IInputFilter>(currentFilters)
-										{
-											new InputFilterAllLower()
-										};
-					_textBoxView.SetFilters(lowerFilter.ToArray());
-
-					break;
-				case CharacterCasing.Upper:
-					var upperFilter = new List<IInputFilter>(currentFilters)
-										{
-											new InputFilterAllCaps()
-										};
-					_textBoxView.SetFilters(upperFilter.ToArray());
-
-					break;
-				case CharacterCasing.Normal:
-					break;
-			}
```

Comment:
Can you validate that visually, the result is the same? Those filters where supposed to change the casing as the user was typing, not after the focus is lost.

Reply:
Yes! I validated at least in the emulator. The filter was giving a strange behavior, mantaining the Upper when you change from Upper to Normal. 
Without the filter, everything worked exactly other platforms. Typed characters turns to the selected casing.

Reply:
https://user-images.githubusercontent.com/100223422/166962656-9a70fd8d-8395-4fc3-8486-a2af8ad67a0d.mp4

Reply:
Excellent! Can you also make a video for the other platforms?

Reply:
**WASM**

https://user-images.githubusercontent.com/100223422/167125387-ae15a1ab-10ed-45ee-a8ef-8581a31e50df.mp4

**SKIA**

https://user-images.githubusercontent.com/100223422/167125220-bcf92dcc-4ea0-450c-b642-18163b7b30c1.mp4


**UWP**

UWP has a little bug: changing the casing will only apply after entering the control.

https://user-images.githubusercontent.com/100223422/167126251-7b932e68-3c26-49df-9221-74e625045b98.mp4


**iOS**


https://user-images.githubusercontent.com/100223422/167129413-5d63965c-9e48-46bd-95db-1433005312ae.mp4


**macOS**

(waiting an XCode update on the mac. I'll edit this comment ASAP including the macOS video)

Reply:
:( 
Debugging macOS produces the expected result.
However, running with all breakpoints disabled, we get the following result:

https://user-images.githubusercontent.com/100223422/167402876-f4a40355-7b15-43ba-bbba-d33c286fabf5.mov